### ISSUE TITLE
Sound

### ISSUE THREAD

**Issue by monza258** [mskian/onesignal-php#1]
Hey, 

Is it possible to include the option to set the SOUND?

**Comment by mskian** [comment]
yes you can refer this  - https://documentation.onesignal.com/docs/customize-notification-sounds#section-4-send-notifications-with-sounds

**Comment by monza258** [comment]
Can you include this for me pls?

**Comment by mskian** [comment]
Sorry After check there Docs Detailedly Custom Notification Sounds is not available for web push API - https://documentation.onesignal.com/docs/features-setup#section-notification-sounds

**Comment by monza258** [comment]
Generally not or only for certain devices? I only need for iOS Devices. 😊

**Comment by mskian** [comment]
I am using web push API on this Script (For Web Push Not available)
For IOS checkout this - https://documentation.onesignal.com/docs/customize-notification-sounds#section-ios
IOS Parameter - https://documentation.onesignal.com/reference#section-appearance

**Comment by monza258** [comment]
I do not know much about myself. I was looking for a way to quickly send push messages. I did not always want to go on the side signal side. The php site is then on my webspace and I can send to all iOS Users with special Sound file.

**Comment by monza258** [comment]
Hey, any options to include the Option for Sound file? pls help me. I love you page. No one can tell me how it will work to include a sound file. I will use this to send a special sound if I have a Breaking news. pls pls pls
